app/lib/main.dart

```diff
@@ -36,20 +38,41 @@ Future<void> main() async {
   // Turn off the # in the URLs on the web
   usePathUrlStrategy();
   AppConfig? appConfig;
+  String initialRoute = Routes.loading;
   try {
     appConfig = await AppConfig.getAppConfig();
+    if (await isAppOutdated(appConfig)) {
+      initialRoute = Routes.appOutdated;
+    }
   } catch (error) {
     // device could be offline
```

Comment:
Add a debug output for analysis. Might also be worth to track the error case to catch malformed `app_min_version` strings.

Reply:
I moved the `isAppOutdated` call outside of the try catch block.

Fixed in https://github.com/hpi-studyu/studyu/pull/576/commits/8d36dc78771d748483bbf3ad22794a95e73dad06